### ISSUE TITLE
Web APIs throws "no-undef" errors

### ISSUE THREAD

**Issue by rowild** [eslint/eslint#14540]
No matter wwhich of the new Web APIs is used, they all throw an error.

```js
let accelerometer = new Accelerometer()

// See error in screenshot 
```

I know that I can use `eslint-disable-next-line`, however, I thought I share that observation here - maybe it is not even a eslint-related problem and I misunderstand something - in which case I'd be happy to learn about my shortcomings.

Thank you!

![Bildschirmfoto 2021-04-28 um 18 24 48](https://user-images.githubusercontent.com/213803/116438733-097cf900-a84f-11eb-97d5-4fbfb84e8c47.png)

**Comment by aladdin-add** [comment]
to allow web APIs, please set the browser env:
```js
{
  "env": {"browser"： true}
}
```

refs: https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments

**Comment by rowild** [comment]
Ah - phantastic! Thank you so much!

**Comment by rowild** [comment]
Actually this does not work. "new Accelerator()" still throws the error shown above... any ideas?

**Comment by aladdin-add** [comment]
I'm not very familiar with `Accelerometer`, is it a new added web api?(maybe it has not been supported yet) 

2 ways:
* decalre it as a global Accelerometer refs: https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals
* use `window.Accelerometer`

**Comment by rowild** [comment]
Adding it as a global works, thanks. :-) - "Accelerometer" and "Gyroscope " (and other) belong to the Sensor API, which is currently only supported well in Chromium Browsers:
https://developer.mozilla.org/en-US/docs/Web/API/Sensor_APIs

Could this be the reason, why eslint does not support it out of the box?
